### DISCUSSION TITLE
static horizontal line hides sensor data

### DISCUSSION THREAD

**Discussion by tanatata** [dbuezas/lovelace-plotly-graph-card#376] [General]
Hi,

I stumbled across plotly to better visualize water levels of rivers in our area than with the default sensor card and plotly improves things a lot!

What I can't figure out is to add a horizontal line which marks critical values for a potential flooding. I tried to do so with a virtual entity, but as soon as the line of the virtual entity is shown the real sensor values vanish. But the values "are there" somehow hidden, even the label of the current value is shown. Any hints how to improve this?

Two examples showing either the virtual entity or the real sensor data, difference os only the `xaxis` of the virtual entity:

![image](https://github.com/dbuezas/lovelace-plotly-graph-card/assets/30723010/c4ed28ca-b09e-426e-b7ce-f9b528427020)

![image](https://github.com/dbuezas/lovelace-plotly-graph-card/assets/30723010/b62f9bfc-6a90-4c69-bc70-c62210c68326)

**Comment by dbuezas** [comment]
It looks like not only the entity disappears, but also the vertical lines of the grid.
To be honest I don't think I tested multiple x axes at all. I suppose that this may break the code in charge of finding out the currently visible time range.

Here are two ways of working around this:

```yaml
- entity: sensor.thing
- entity: ''
   y: [280,280]
   x: $ex get('visible_range')
```

Or use shapes (this is a lot better)
https://github.com/dbuezas/lovelace-plotly-graph-card/discussions/45#discussioncomment-2768991

**Reply by tanatata** [reply]
Thanks, works for me!
